.github/workflows/stale.yml

```diff
@@ -1,19 +1,48 @@
-name: "Mark and close stale issues"
+name: "Label and close stale issues & PRs"
+
 on:
   schedule:
   - cron: "0 0 * * *"
+  workflow_dispatch:
+    inputs:
+      debug:
+        description: "Run in debug mode (dry run)"
+        type: boolean
+        default: true
 
 jobs:
   stale:
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/stale@v9
       with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 30 days'
-        days-before-stale: 30
-        days-before-close: 30
-        close-issue-message: 'Issue closed due to inactivity.'
+        repo-token: ${{secrets.GITHUB_TOKEN}}
+        debug-only: ${{inputs.debug || false}}
+        days-before-stale: 90
+        days-before-close: 60
```

Comment:
~~I think we should flip these.  The issue / PR should become stale before closed.~~
NVM - after reading how those values work, sorry about the noise.